articles/quickstart.md

```diff
@@ -1,163 +1,129 @@
 1. One or more quantum algorithms, implemented using the Q# quantum programming language.
-2. A classical program, implemented in a classical programming language like Python or C#, 
-  that serves as the main entry point and will invoke Q# operations 
-  when it wants to execute a quantum algorithm.
+1. A host program, implemented in a programming language like Python or C# that serves as the main entry point and invokes Q# operations to execute a quantum algorithm.
 
-This guide will walk you through how to create applications using Q#, and using C# as the classical component in Visual Studio or Visual Studio Code.
+#### [Python](#tab/tabid-python)
 
-## Creating a Bell State in Q#
+1. Choose a location for your application
 
-Now that you’ve [installed the Microsoft Quantum Development Kit](xref:microsoft.quantum.install), let’s write your first quantum application.
-We'll start with the simplest program possible and build it up to demonstrate [quantum superposition](https://en.wikipedia.org/wiki/Quantum_superposition) 
-and [quantum entanglement](https://en.wikipedia.org/wiki/Quantum_entanglement). We will start with a qubit in a 
-basis state $\ket{0}$, perform some operations on it and then measure the result.
+1. Create a file called `Bell.qs`. This file will contain your Q# code.
 
-The instructions on the page are written for both the command line and Visual Studio.
-For Visual Studio, it assumes you have installed the 
-[Microsoft Quantum Development Kit Visual Studio extension](https://marketplace.visualstudio.com/items?itemName=quantum.DevKit).
-If you are using a different development environment, like Visual Studio Code, please 
-follow the instructions to call the .NET Core SDK from the command line.
+1. Create a file called `host.py`. This file will contain your python host code.
 
-### Step 1: Create a Project and Solution
+#### [C# Command Line](#tab/tabid-csharp)
 
-#### [Command Line / Visual Studio Code](#tab/tabid-vscode)
+1. Create a new Q# project:
 
-Run the following commands in your favorite command line (e.g.: PowerShell or Bash):
+    ```bash
+    dotnet new console -lang Q# --output Bell
+    cd Bell
+    ```
 
-```bash
-$ dotnet new console -lang Q# --output Bell
-$ cd Bell 
-$ code . # To open in Visual Studio Code.
-```
+    You should see a `.csproj` file, a Q# file called `Operation.qs`, and a host program file called `Driver.cs`
+
+1. Rename the Q# file
+
+    ```bash
+    mv Operation.qs Bell.qs
+    ```
 
 #### [Visual Studio](#tab/tabid-vs2019)
 
-- Open up Visual Studio.
-- Go to the `File` menu and select `New` > `Project...`.
-- In the project template explorer, under `Installed` > `Visual C#`, select the `Q# Application` template.
-- Make sure you have `.NET Framework 4.6.1` selected in the list at the bottom of the `New Project` dialog box.
-- Give your project the name `Bell`.
+1. Create a new project
 
-***
+   * Open Visual Studio
```

Comment:
Some of these bullet points end in periods, while others don't? I'm not sure of the best style, though.